### PR TITLE
Edit developer guide

### LINKED ISSUE #68: Edit Developer Guide to add Use Cases for corresponding assigned features

### PR DESCRIPTION
**Pull request by SelwynAng**
* Edited developer guide by making these changes:
  * Add Use Case for Delete a member 
  * Add Use Case for List all members
  * Add Use Case for List all events

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -285,29 +285,65 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Delete a member**
 
 **MSS**
```
Comment (wr1159) [inline docs/DeveloperGuide.md:290]: What do you think about adding a `guarantees` in between here?
```suggestion
**Use case: UC01 - Delete a member**

**Guarantees: MSS -> Specified event will be deleted.**

**MSS**
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -285,29 +285,65 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    Use case ends.
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. CCACommander shows an error message.
 
       Use case resumes at step 2.
 
+**Use case : UC02 - List all members**
```
Comment (wr1159) [inline docs/DeveloperGuide.md:313]: Should we keep it to 1 `list` use case or can we separate them into both members and events? I think both should be okay

**Review by wr1159 — COMMENTED** [review]
Left 2 suggestions about 
* adding guarantees in use case
* combining 2 lists command

Other than that LGTM

I will merge this once you let me know what you think about the suggestions, we don't have to follow them, just throwing out ideas.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (SelwynAng) [reply docs/DeveloperGuide.md:290]: Sounds great, I will implement that

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (SelwynAng) [reply docs/DeveloperGuide.md:313]: I will keep them separated.

**Comment by SelwynAng** [comment]
Will close this pull request and create a new one incorporating the suggestions given by @wr1159 from this pull request.

**Comment by wr1159** [comment]
I think we can keep the same pull request and push the updated changes to your `edit-developer-guide` branch so there's no need to open a new pull request. It will preserve the previous comments for easier following too.

**Pull request closed without merging**